flink-connector-aws-base/src/main/java/org/apache/flink/connector/aws/util/AWSClientUtil.java

```diff
@@ -194,7 +196,7 @@ S createAwsSyncClient(
         final ClientOverrideConfiguration overrideConfiguration =
                 createClientOverrideConfiguration(
                         clientConfiguration,
-                        ClientOverrideConfiguration.builder(),
+                        clientOverrideConfigurationBuilder,
```

Comment:
Let's keep the old method to keep backwards compatibility

Reply:
my bad, fixed this in next revision